worker_server.py

```diff
@@ -14,6 +15,7 @@
 
 class WorkerServer(api_pb2_grpc.WorkerApiServicer):
     def __init__(self):
+        print(f"IN init {WORKER_PORT}")
```

Comment:
nit: change to log

Reply:
that log is actually not needed - removed it